### DISCUSSION TITLE
Documentation Improvement: Publishing views

### DISCUSSION THREAD

**Discussion by umangapathak999** [filamentphp/filament#3521] [Ideas]
On the documentation page:
**Admin Panel > Appearance**
[https://filamentphp.com/docs/2.x/admin/appearance](https://filamentphp.com/docs/2.x/admin/appearance)

I could not find the step to publish the views publish command that is required to make changes to filament appearance. Please add the following command to the documentation.

`php artisan vendor:publish --tag=filament-views`

Since I could not find the documentation edit option, I am opening the issue. Hope it helps the community.

By the way, love filament and it definitely has made my work easier and helps me focus on the actual logic rather than creating admin panel and how it looks and functions.

**Comment by haztakki** [comment]
It might be more appropriate to ask here:
https://github.com/filamentphp/filament/discussions/876

**Comment by ryangjchandler** [comment]
We don't actively encourage people to publish views because it potentially opens up a bunch of support issues that are out of our control.

**Comment by danharrin** [comment]
As Ryan said, please do not publish views.

https://filamentphp.com/blog/publishing-views-in-laravel

We have many filament- CSS classes you can use to style Filament using your own theme CSS file. This is much safer as these wont change, and your views wont become outdated.

**Reply by Razor30** [reply]
this is a mess ... 10 hours and i could not modify the app.blade.php to display the information as i want. please provide detailed information how can you push the scripts to the footer and how can you change the vite directives.
